app/pb_kits/playbook/pb_owner/_owner.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= content_tag(:div,
+    id: object.id,
+    data: object.data,
+    class: object.classname("pb_owner")) do %>
+      <div class="owner_first">
+        <div> <%= object.first_name %> </div>
+      </div>
+      <div class="owner_last">
+        <div> <%= object.last_name %> </div>
```

Comment:
Can you add `pb_owner_last ` as a classname prop in the kit class? Also lets remove this wrapper.

Reply:
Corrected!